frontend/src/components/accountant/DocxPreview.jsx

```
@@ -1,219 +1,219 @@
+              .docx-content h1 { font-size: 24px; margin-top: 24px; margin-bottom: 8px; }
+              .docx-content h2 { font-size: 20px; margin-top: 20px; margin-bottom: 8px; }
+              .docx-content h3 { font-size: 16px; margin-top: 16px; margin-bottom: 8px; }
+              .docx-content p { margin-bottom: 16px; }
+              .docx-content ul, .docx-content ol { margin-bottom: 16px; padding-left: 24px; }
+              .docx-content table { border-collapse: collapse; margin-bottom: 16px; }
+              .docx-content table, .docx-content th, .docx-content td { border: 1px solid #ddd; }
+              .docx-content th, .docx-content td { padding: 8px; }
+              .docx-content img { max-width: 100%; height: auto; }
+            `}
+          </style>
+          <div dangerouslySetInnerHTML={{ __html: htmlContent }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)